corehq/apps/app_manager/util.py

```diff
@@ -119,37 +119,34 @@ def split_path(path):
 
 
 def save_xform(app, form, xml):
```

Comment:
Yep, that's what was happening. I removed the check for `ShadowForm` since it sounds like that's not important here.